Testing/ImageTests/ImageTests.cpp

```diff
@@ -19,6 +19,43 @@ TEST(ImageTests, resample_test) {
   ASSERT_TRUE(image.compare_equal(ground_truth));
 }
 
+TEST(ImageTests, padimage_test) {
+  std::string test_location = std::string(TEST_DATA_DIR) + std::string("/padimage/");
+
+  Image image(test_location + "1x2x2.nrrd");
+  image.padimage(30, 0.0);
+ 
+  Image ground_truth(test_location + "padimage_baseline.nrrd");
+
+  ASSERT_TRUE(image.compare_equal(ground_truth));
+}
+
+TEST(ImageTests, padimage_identity_test) {
+  std::string test_location = std::string(TEST_DATA_DIR) + std::string("/padimage/");
+
+  Image image(test_location + "1x2x2.nrrd");
+  image.padimage(0, 0.0);
+  Image ground_truth(test_location + "1x2x2.nrrd");
+
+  ASSERT_TRUE(image.compare_equal(ground_truth));
+}
+
+TEST(ImageTests, binarize_test) {
+
+std::string test_location = std::string(TEST_DATA_DIR) + std::string("/binarize/");
+
+Image image(test_location + "1x2x2.nrrd");
```

Comment:
As we were learning the hard way before, let's limit these tests to one per operation. Or we can figure out some way to indicate to the developer whose test fails which part of the test that failed (maybe googletests has some way to indicate "subtests" within a test).